src/makeCalculator.js

```diff
@@ -4,7 +4,39 @@
  * @return {object}
  */
 function makeCalculator() {
-  // write code here
+  const calculator = {
+    result: 0,
+
+    operate(callback, operand) {
+      this.result = callback(operand, this.result);
+
+      return this;
+    },
+
+    add(operand, result) {
+      return result + operand;
```

Comment:
As per the checklist, use arithmetic operators with assignment operator `=`. Instead of `return result + operand;`, use `return result += operand;`. This applies to the `add`, `subtract`, `multiply`, and `divide` methods.